src/main/java/com/google/cloud/genomics/utils/Contig.java

```diff
@@ -108,7 +110,41 @@ public SearchReadsRequest getReadsRequest(String readGroupSetId) {
         .setEnd(end);
   }
 
-  public static Iterable<Contig> parseContigsFromCommandLine(String contigsArgument) {
+  public StreamReadsRequest getStreamReadsRequest(String readGroupSetId) {
+    return StreamReadsRequest.newBuilder()
+        .setReadGroupSetId(readGroupSetId)
+        .setReferenceName(referenceName)
+        .setStart(start)
+        .setEnd(end)
+        .build();
+  }
+
+  private List<Contig> getShards() {
```

Comment:
As a matter of taste, I much prefer when all the public methods are first in the file, then all the private ones at the bottom. The reason is that most people are just users of the class, so they can stop reading after the last public method.